CMakeLists.txt

```diff
@@ -318,7 +334,7 @@ find_package(gflags COMPONENTS shared)
 find_package(glog REQUIRED)
 
 set_source(fmt)
-resolve_dependency(fmt 8.0.1)
```

Comment:
This seems a bit unrelated. What's this change about?

Reply:
The torcharrow image had system fmt installed which was the old and incompatible version 6.x. This change sets a minimum version for fmt and prevents `resolve_dependency`  from picking up old system versions and instead build fmt from source, I used 8.0.1 as that is what folly uses (and the first version that has the internal v8 namespace)